internal/store/engine_state.go

```diff
@@ -584,11 +584,11 @@ func (ms *ManifestState) StartedFirstBuild() bool {
 	return !ms.CurrentBuild.Empty() || len(ms.BuildHistory) > 0
 }
 
-func (ms *ManifestState) MostRecentPod() Pod {
+func (ms *ManifestState) MostRecentPod() filewatches.Pod {
```

Comment:
hmm now that the v1alpha package contains way more than just filewatches, we should probably not alias it to this (/in general, should always refer to it as `v1alpha` for consistency)

Reply:
Ha that's what happens when I let IntelliJ do the refactor for me! I'll remove this alias, thanks for catching.

But yeah, to the broader point, I guess the only time it's probably "appropriate" to alias the import is within a type-specific reconciler/controller package, so I should really never have used it here in the first place. And even within the reconciler package... 🤷 probably not worth it?

Reply:
'tis gone as of ee3a48e38